Makefile

```diff
@@ -172,10 +171,9 @@ clean:
 distclean:
 	@rm -rf node_modules
 
-# create list of translations, saved as `./calypso-strings.php`
+# create list of translations, saved as `./calypso-strings.pot`
 translate: node_modules $(CLIENT_CONFIG_FILE)
-	@CALYPSO_ENV=stage $(BUNDLER)
-	@CALYPSO_ENV=stage $(LIST_ASSETS) | xargs $(I18N_CALYPSO) --format php --output-file ./calypso-strings.php --array-name calypso_i18n_strings
+	@CALYPSO_ENV=stage $(I18N_CALYPSO) --format pot --output-file ./calypso-strings.pot $(JS_FILES)
```

Comment:
So I'm trying to follow the old `server/bin/list-assets.js` . At the very least, I don't think you need `CALYPSO_ENV=stage` here anymore, as this was driving which assets file was loaded in the `list-assets` script. What I haven't been able to determine yet is whether those files were generated for the sole purpose of i18n and can be further cleaned up.